extension/server/src/providers/linter/documentFormatting.ts

```diff
@@ -23,7 +23,8 @@ export default async function documentFormattingProvider(params: DocumentFormatt
 				// Need to fetch the docs again incase comments were added
 				// as part of RequiresProcedureDescription
 				docs = await parser.getDocs(document.uri, document.getText(), {
-					ignoreCache: true
+					ignoreCache: true,
+					withIncludes: true
```

Comment:
Why was this added here? I recall as the second time around, we don't need to have the includes again.

Reply:
I thought the `IncorrectVariableCase` option didn't work with templates defined in copybooks unless I set `withIncludes` but have since successfully tested it without this change so I'll back this out.

Reply:
I take it back. The linter shows the problem within the UI but the `Format Document` option does not fix references to template fields in a copybook.

For example.

![image](https://github.com/codefori/vscode-rpgle/assets/7809440/8d328ba1-55cf-4c10-b436-d64e17a642c5)

The `customer_name_t` is defined in the `CUSTOMER_D` copybook as `CUSTOMER_NAME_t`. I can perform an individual `Quick Fix` on that one issue but `Format Document` only fixes the `user_name_t` reference.

I've reinstated the change.

[CUSTOMER_D.rpgle.txt](https://github.com/codefori/vscode-rpgle/files/13979354/CUSTOMER_D.rpgle.txt)
[LINT01.rpgle.txt](https://github.com/codefori/vscode-rpgle/files/13979355/LINT01.rpgle.txt)